engines/pytorch/pytorch-engine/src/main/java/ai/djl/pytorch/engine/PtModel.java

```diff
@@ -197,7 +197,7 @@ public Trainer newTrainer(TrainingConfig trainingConfig) {
             // Unfreeze parameters if training directly
             block.freezeParameters(
                     false,
-                    p -> p.getType() != Type.RUNNING_MEAN && p.getType() != Type.RUNNING_VAR);
+                    param -> param.getType() != Type.RUNNING_MEAN && param.getType() != Type.RUNNING_VAR);
```

Comment:
short parameter name in lambda is very common. I don't think this change is necessary.